src/components/CreateConfig.vue

```diff
@@ -52,38 +65,81 @@
       placeholder="config name"
       :rules="[validateConfigName]"
   ></v-text-field>
+  <v-dialog v-model="showErrorDialog" @click:outside="resetDialog()">
+    <v-card color="danger">
+      <v-card-title>Couldn't load config</v-card-title>
+      <v-card-text>
+        <pre><code>{{ errors }}</code></pre>
+      </v-card-text>
+      <v-card-actions>
+        <v-btn color="primary" @click="resetDialog()">OK</v-btn>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
 </template>
 
 <script setup lang="ts">
 import {computed, ref} from "vue";
 import {useEvbcStore} from "@/store/evbc";
 import {storeToRefs} from "pinia";
+import yaml from "js-yaml";
+import Ajv from "ajv";
+import {EverestConfig} from "@/modules/evbc";
+import {urlToPublicAsset} from "@/utils";
 
 enum ComponentStates {
-    DEFAULT,
-    ASK_USER_FOR_CONFIG_NAME,
+  DEFAULT,
+  ASK_USER_FOR_CONFIG_NAME,
   }
```

Comment:
```suggestion
}
```

Should be indented like that too I guess?